### PR TITLE
GetView and SetView could be a Property 

### PR DESCRIPTION
**Pull request by Kuinox**
Moved GetView and SetView to a Property in:
IRenderTarget
And the implementations:
RenderTexture
RenderWindow

### CODE REVIEW

File: ITI.SFML.Graphics/RenderWindow.cs [hunk 1]
```diff
@@ -293,23 +293,13 @@ public void Clear( Color color )
...
-        /// </summary>
-        /// <param name="view">New view.</param>
-        public void SetView( View view )
-        {
-            sfRenderWindow_setView( CPointer, view.CPointer );
-        }
-
         /// <summary>
-        /// Returns the current active view.
+        /// Get or Set the current active view
```
Comment (olivier-spinelli) [inline ITI.SFML.Graphics/RenderWindow.cs]: Gets or sets (this is the standard).

File: ITI.SFML.Graphics/IRenderTarget.cs [hunk 2]
```diff
@@ -17,17 +17,9 @@ public interface IRenderTarget
...
-        /// <returns>The current view</returns>
-        View GetView();
 
-        /// <summary>
-        /// Changes the current active view.
-        /// </summary>
-        /// <param name="view">New view</param>
-        void SetView( View view );
+
+        View View { get; set; }
```
Comment (olivier-spinelli) [inline ITI.SFML.Graphics/IRenderTarget.cs]: What's the difference between DefaultView and View/(Get/SetView)?
Isn't there a reason for this 2 different API point?

**Review by olivier-spinelli — CHANGES_REQUESTED** [review]

File: ITI.SFML.Graphics/IRenderTarget.cs (hunk 2 above)
Comment (Kuinox) [reply ITI.SFML.Graphics/IRenderTarget.cs]: SFML docs say: 
Get the default view of the render target.

The default view has the initial size of the render target, and never changes after the target has been created.

**Comment by olivier-spinelli** [comment]
Ok : this needs to be reported in the DefaultView’s comment 😊.

**Comment by Kuinox** [comment]
Ah, j'ai fais le même changement de mon coté :<

**Pull request merged**